rst_files/version_control.rst

```diff
@@ -280,7 +285,7 @@ GitHub will give us the following window
 * The warning symbol next to the file indicates the existence of a merge conflict 
 * The viewer tries to show us the discrepancy (I changed the word repository to repo, but someone else tried to change it to "repo" with quotes)
 
-To fix the conflict, we can go into a text editor (such as Atom or VS Code). Here's an image of what we see in Atom 
+To fix the conflict, we can go into a text editor (such as Atom or VS Code)
```

Comment:
```suggestion
To fix the conflict, we can go into a text editor (such as Atom)
```

Guess I'll just eliminate the mention of VS Code here, give them less options `:-)`

Reply:
Sure